### DISCUSSION TITLE
Feature: No autocomplete in VS Code snippets

### DISCUSSION THREAD

**Discussion by Sti2nd** [community/community#8752] [Copilot Conversations]
Github Copilot conflicts with all snippets in VS Code (that have more than one tab stop) because both use the <kbd>Tab</kbd> key.

The example which I just encountered was using the `import statement` snippet (for JS/TS) that is built into VS Code. 

Steps to reproduce:

1. Write `imp` in a file and trigger autocomplete
2. Choose `import statement`
3. Begin to write the name of a package, e.g. `r` and Copilot will hopefully kick in.
![image](https://user-images.githubusercontent.com/6373430/145719486-3376dd39-e33d-49ca-8ef3-4acfa3638e6c.png)
4. Problem! It is impossible to tab to the second tab stop. If one writes out `react` the Copilot suggestion will still be there, and if one press <kbd>ESC</kbd> the second tab stop disappears.

I think it would be great if Copilot could avoid triggering when I am in the middle of filling out a snippet, for example until I reach the last tab stop. Alternatively, if it simply was possible to dismiss Copilot without dismissing the tab stops that would also work.

There is already created a discussion about user snippets here https://github.com/github/feedback/discussions/7012

**Comment by otkrickey** [comment]
I'm having the same conflict in my environment and it's a bit annoying.
